src/common/containers/my-repos.js

```diff
@@ -15,10 +16,20 @@ class MyRepos extends Component {
         />
         <BetaNotification />
         <UserAvatar />
-        <RepositoriesHome />
+        <RepositoriesHome hasJustSignedIn={this.props.hasJustSignedIn}/>
       </div>
     );
   }
 }
 
-export default MyRepos;
+MyRepos.propTypes = {
+  hasJustSignedIn: PropTypes.bool
+};
+
+function mapStateToProps(state, ownProps) {
+  return {
+    hasJustSignedIn: ownProps.location.query['sign-in'] !== undefined
```

Comment:
That's where React Router passes current location to a component rendered by given route.

It's only visible on container level, because that's a component [referenced in the route](https://github.com/canonical-ols/build.snapcraft.io/blob/master/src/common/routes.js#L18).